proj/src/sub2/bin/foo_main.ml

```diff
@@ -25,7 +25,7 @@ let subcommands = [
 
 let help () =
   let subcommand_names =
-    String.concat "\n" (List.map (fun (name, f) -> "  " ^ name) subcommands)
+    String.concat "\n" (List.map (fun (name, _f) -> "  " ^ name) subcommands)
```

Comment:
Hmm, I don't like this style very much.

Reply:
My old self is not convinced. I've heard this style is mandatory in Rust... but I don't see what purpose it serves.

Reply:
Which style? Turning on warning-as-error in development mode?

Reply:
I see the value of `--dev` giving us warnings that are relevant to the developer but might be useless and confusing to the simple user.

What I don't like is having to prefix unused variables with an underscore, with the exception of let bindings.